SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -70,7 +72,6 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	template <typename T, typename... ArgsType>
 	void QueueOutgoingMessage(ArgsType&&... Args);
 
-private:
```

Comment:
Removing the private?

Reply:
Was a duplicate label, there is another above on line 59

Reply:
gotcha